Fix dead links and stale browser asides in SVG tutorials (#44998)
mdn/content: files/en-us/web/svg/tutorials/svg_from_scratch/getting_started/index.md

Reviewer: added "following"
```suggestion
Now that you have an idea of how to create basic SVG files, the next stage is to upload them to a web server. There are some gotchas at this stage though. For normal SVG files, servers should send the following HTTP headers:
```

Reviewer: same, added "following"
```suggestion
For gzip-compressed SVG files, servers should send the following HTTP headers:
```

SVG files come in two flavors. Normal SVG files are text files containing SVG markup. The recommended filename extension for these files is ".svg" (all lowercase).

> Due to the potentially massive size SVG files can reach when used for some applications (e.g., geographical applications), the SVG specification also allows for gzip-compressed SVG files. The recommended filename extension for these files is ".svgz" (all lowercase). A gzip-compressed SVG only loads correctly when the server sends the right `Content-Encoding` header (see below), so only use `.svgz` when you know your web server is configured to serve it correctly.

Reviewer: ```suggestion
Due to the potentially massive size SVG files can reach when used for some applications (e.g., geographical applications), the SVG specification also allows SVG files to be [gzip-compressed](/en-US/docs/Glossary/gzip_compression). The recommended filename extension for these files is ".svgz" (all lowercase). A gzip-compressed SVG loads correctly only when the server sends the correct `Content-Encoding` header (see the next section), so use `.svgz` only when you know your web server is configured to serve it correctly.
```

Vary: Accept-Encoding
```

> You can check that your server is sending the correct HTTP headers with your SVG files by opening your browser's developer tools and inspecting the response headers in the [Network panel](https://firefox-source-docs.mozilla.org/devtools-user/network_monitor/index.html#headers). Look at the HTTP response headers for one of your SVG files. If you find that your server is not sending the headers with the values given above, then you should contact your web host or, if you manage the server yourself, update its configuration to serve SVG with the headers shown above.

Reviewer: - We can link "developer tools" to https://developer.mozilla.org/en-US/docs/Glossary/Developer_Tools and cite that Network Panel link as an example in Firefox.
- "given above", "shown above" can be replaced with "shown"
